src/server/auth/server/util.go

```diff
@@ -27,8 +46,8 @@ func CheckRepoIsAuthorizedInTransaction(txnCtx *txnenv.TransactionContext, r str
 	return nil
 }
 
-// CheckRepoIsAuthorized returns an error if the current user (in 'pachClient') has
-// authorization scope 's' for repo 'r'
+// CheckRepoIsAuthorizedInTransaction returns an error if the current user doesn't have
+// the permissions in `p` on the repo `r`
 func CheckRepoIsAuthorized(pachClient *client.APIClient, r string, p ...auth.Permission) error {
```

Comment:
Nit: the method itself is still `CheckRepoIsAuthorized` (rather than `CheckRepoIsAuthorizedInTransaction`); maybe a stray find & replace?